public_suffix_list.dat

```diff
@@ -11271,6 +11271,9 @@ zakopane.pl
 pantheon.io
 gotpantheon.com
 
+// Peplink | Pepwave : http://peplink.com/
+mypep.link
```

Comment:
@steveleung can you please add a line including a valid email address we can use in case of issue?

```
// Peplink | Pepwave : http://peplink.com/
Submitted by FirstName LastName <example@example.com>
```